samples/apps/copilot-chat-app/README.md

```diff
@@ -55,6 +55,8 @@ First, let’s set up and verify the back-end API server is running.
        ```bash
        dotnet user-secrets set "AzureSpeech:Key" "MY_AZURE_SPEECH_KEY" 
        ```
+     - In the .env file that you create below, set REACT_APP_ENABLE_SPEECH to true.  
+       ```REACT_APP_ENABLE_SPEECH=true```
```

Comment:
I thought we were going to autodetect this

Reply:
Well we were auto-detecting it prior to this by only creating the input symbol if we found a valid speech resource. But when there isn't a key it would throw an error, that doesnt affect the rest of the app but does look bad for a reference app. So we decided to just hide it behind a bool for now.

Reply:
frontend => request to our backend, asking for speech config

    if (MY_AZURE_SPEECH_KEY is set)
        backend uses AAD token to get a Speech token
        return 200 + Config
    else:
        return 404 + "NOT AVAILABLE"